cmd/lib/keypair.js

```diff
@@ -0,0 +1,53 @@
+const bip39 = require('bip39');
+const bip32 = require('bip32');
+const secp256k1 = require('secp256k1');
+const {
+    bech32
+} = require('bech32');
+const createHash = require('create-hash');
+
+function createFromMnemonic(mnenonic, hdPath) {
+    if (typeof mnemonic === 'undefined') {
```

Comment:
argument is different from checked variable
createFromMnemonic(mnenonic, hdPath)
mnemonic === 'undefined']

**mnenonic != mnemonic**